storage.tsx

```diff
@@ -25,8 +54,9 @@ class Storage {
   }
 
   // Create and populate database tables
-  createTable() {
-    {
+  async createTable(downloadCompletedCallback: Function) {
+    downloadNMWCodes(NMWSTANDARD).then((nmwTable) => {
+      console.log(nmwTable);
```

Comment:
Could we remove this log before publishing?

Reply:
Good catch, I've removed it.